src/REST/Client.php

```diff
@@ -19,6 +25,11 @@ public static function instance() : Client
         return static::$instance;
     }
 
+    /**
+     * Client constructor.
+     * @param string $jira_url
+     * @param string $api_prefix
+     */
     public function __construct(
         $jira_url   = ClientRaw::DEFAULT_JIRA_URL,
         $api_prefix = ClientRaw::DEFAULT_JIRA_API_PREFIX
```

Comment:
just to think about: maybe it is good to make this arguments typed.
`string $jira_url = ClientRaw::...`

Reply:
You're absolutely right
But the purpose of this request is different